app/models/post.rb

```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :comments, foreign_key: 'post_id'
   has_many :likes, foreign_key: 'post_id'
 
+  validates :Title, presence: true
```

Comment:
Please add the proper validation in order for your **title** to not exceed 250 characters. Remember that validation provides accuracy, details, and clarity to your data.

Reply:
Thank you. the issue has been fixed

Reply:
![review](https://user-images.githubusercontent.com/63972670/146656531-998151de-b5d3-4922-a4f9-925e74b0805d.JPG)


![Areview](https://user-images.githubusercontent.com/63972670/146656543-d918bd90-5d9e-4e36-a476-133ea6cf23c5.JPG)

Reply:
Migration works well at my end and the server is running actually